cfme/tests/infrastructure/test_host_cpu_resources.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+from utils import testgen
+from cfme.web_ui import Quadicon, toolbar
+from cfme.infrastructure.host import Host
+from cfme.web_ui import InfoBlock
+from cfme.fixtures import pytest_selenium as sel
+from cfme.configure.tasks import is_host_analysis_finished
+from utils.wait import wait_for
+
+
+def pytest_generate_tests(metafunc):
+    argnames, argvalues, idlist = \
+        testgen.infra_providers(metafunc, required_fields=["ssh_credentials"])
+    testgen.parametrize(metafunc, argnames, argvalues, ids=idlist,
+                        scope="module")
+
+
+def get_integer_value(x):
+    return int(x.split(' ')[0])
```

Comment:
This is something I suggested to @rrasouli because he was trying to get the numeric value but the .value cannot process units.

I wonder if I did some system that would parse and represent the units in meaningful way? With comparison and everything?